superset/assets/src/dashboard/components/FilterIndicatorsContainer.jsx

```diff
@@ -0,0 +1,162 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+/* eslint-disable camelcase */
+/* eslint-disable no-confusing-arrow */
```

Comment:
which arrow is confusing? Can we fix the lint error instead? If not, can we just disable it for the line that's problematic?

Reply:
it was from this line:
```          
.filter((indicator, index) =>
            showExtraIndicatorsInGroup
              ? index < FILTER_INDICATORS_DISPLAY_LENGTH
              : true,
          )
```
arrow function doesn't like ternary operator.
I fixed by using if else.